src/dcat-us/_generate-distributions.ts

```diff
@@ -53,6 +58,16 @@ function isLayer (dataset: any) {
   return /_/.test(dataset.id);
 }
 
+function isProxiedCSV(dataset: any) {
+  const item = datasetToItem({
+    id: dataset.id,
+    attributes: dataset
+  } as DatasetResource);
+  const requestOptions: IHubRequestOptions = { isPortal: false };
+  
+  return !!getProxyUrl(item, requestOptions);
```

Comment:
Thinking out loud...

This set of conversions is clever, but squishy. We're basically taking a V3 API response, making it look sorta like a dataset, then making that sorta look like an item, then throwing it into the `getProxyUrl` method which expects a full item. 

We can do this because we happen to know about the internals of `getProxyUrl` enough to know that this will work.

That said, if it works—great. A regression here would be low-impact and you have a test to validate it anyways.

One question though—why not just use [isProxiedCSV](https://github.com/Esri/hub.js/blob/64dfd02f958ca58d43e7eaf0a467bc59859a42d6/packages/common/src/content/_internal.ts#L92) instead of  `getProxyUrl`?

Reply:
Okay—I see now that `isProxiedCSV` isn't part of the Hub.js public API. I'm not sure why—but you probably had a reason.

Reply:
@andrewctate thanks for the review! 

> Okay—I see now that `isProxiedCSV` isn't part of the Hub.js public API. I'm not sure why—but you probably had a reason.

That was my original plan, but Tom is really trying to minimize hub.js' public footprint. I discussed it with him, but ultimately he asked for us to do it this way 🤷‍♂️

> This set of conversions is clever, but squishy.

I know it's not ideal. Tom is doing a massive refactor for getting IHubContents that we'll probably use in the near future. The result of that refactor will already have the proxied csv stuff baked in 🙌